faculty/datasets/path.py

```diff
@@ -93,3 +78,30 @@ def project_parent_directories(project_path):
         directories.append("/".join(parts[:i_last]) + "/")
 
     return directories
+
+
+def project_parent_directory(project_path):
```

Comment:
GitHub won't let me comment there, but it seems we're not using `project_parent_directories` (above) any more?